manifests/z/Zanvok/Browsio/98.21.12.26/Zanvok.Browsio.locale.en-US.yaml

```
@@ -0,0 +1,34 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Zanvok.Browsio
+PackageVersion: 98.21.12.26
+PackageLocale: en-US
+Publisher: Zanvok Corporation
+PublisherUrl: https://sites.google.com/view/zanvokcorporation/
+PublisherSupportUrl: https://sites.google.com/view/zanvokhelp/
+PrivacyUrl: https://sites.google.com/view/zanvokhelp/
+Author: Gautham Nair
+PackageName: Browsio
+PackageUrl: https://sites.google.com/view/browsio/
+License: MPL-2.0 License
+LicenseUrl: https://www.mozilla.org/en-US/MPL/2.0/
+Copyright: 2022 Gautham Nair Zanvok Corporation
+CopyrightUrl: https://sites.google.com/view/zanvokcorporation/
+ShortDescription: A Simple & Useful Tabbed Browser
+Description: Browsio is written in Python3, and it is intended to run for fast and emergency purposes, as it starts up fast...
+Moniker: Zanvok Browsio
+Tags:
+- Browser
+- Browsio
+- Browsee
+- Py Browser
+- Python
+- Python3
+- Zanvok
+- Zanvok Corporation

```

```suggestion
Tags:
- browser
- browsee
- py-browser
- python
- python3
- zanvok-corporation
```
Tags should be lower-cased and use `-` instead of spaces